src/Cognito/Provider.php

```diff
@@ -86,13 +85,7 @@ protected function getUserByToken($token): array
      */
     protected function mapUserToObject(array $user)
     {
-        return (new User())->setRaw($user)->map([
-            'id'       => $user['sub'],
-            'nickname' => $user['nickname'],
-            'name'     => trim(Arr::get($user, 'given_name', '').' '.Arr::get($user, 'family_name', '')),
-            'email'    => $user['email'],
```

Comment:
Why not default these to null instead of removing them completely? this will be a BC break.

Reply:
Good call, I was thinking that having all of the fields in one user array may be cleaner but I think this is ok, nulls added. Thanks :)